src/Conversion/AutoConvert.ts

```diff
@@ -0,0 +1,73 @@
+import Encoding from './Encoding';
+
+class AutoConvert {
+  // Forced encoding can be useful when converting an entire string
+  // or simply dealing with binary which has no leading zeros
+  public static convertCharacter(input: string, forcedEncoding?: Encoding) {
+    let encoding = null;
+    if (!forcedEncoding) {
+      encoding = this.determineCharacterEncoding(input);
+    } else {
+      encoding = forcedEncoding;
+    }
+
+    // Assume that Latin characters should remain
+    if (encoding === Encoding.Latin) {
+      return input;
+    }
+
+    const baseTen = parseInt(input, 10);

```

Comment:
I think `Number.parseInt` is better since it's more strict.  If it doesn't contain only an integer then it will return `NaN`.

Reply:
Looks like `Number.parseInt` is an ES2015 function, so it won't work here.